srcs/juloo.keyboard2/KeyValue.java

```diff
@@ -190,6 +192,11 @@ private static void addEventKey(String name, String symbol, int event, int flags
     addCharKey('#', KeyEvent.KEYCODE_POUND);
     addCharKey('(', KeyEvent.KEYCODE_NUMPAD_LEFT_PAREN);
     addCharKey(')', KeyEvent.KEYCODE_NUMPAD_RIGHT_PAREN);
+    // Can't combine here, right?
+    // Is there another way to have these reachable directly?
+    // addCharKey('ä', KeyEvent.VK_DEAD_DIAERESIS + KeyEvent.VK_A)
+    // addCharKey('ö', KeyEvent.VK_DEAD_DIAERESIS + KeyEvent.VK_O)
+    // addCharKey('ü', KeyEvent.VK_DEAD_DIAERESIS + KeyEvent.VK_U)
```

Comment:
We don't need keycodes for these characters, so no need to define them here.

Reply:
I thought it was needed for the Capital letters - to be able to combine with Shift to get ÄÖÜ - is that incorrect?

Reply:
It's not needed. Character modifications are first handled by Java's functions and when that's not enough, the rules are in `KeyModifier.java`.